pixl_export/tests/test_export_processing.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+These tests require executing from within the export-api container with the dependent
+services being up
+    - pixl postgres db
+    - emap star
+"""
+
+from __future__ import annotations
+
+import dataclasses
+from logging import getLogger
+from typing import Any
+
+import pytest
+
+pytest_plugins = ("pytest_asyncio",)
+
+logger = getLogger(__name__)
+
+
+@dataclasses.dataclass
+class MockResponse:
+    """Mock response object for get or post."""
+
+    status_code = 200
+    content: str | None
+    text: str | None
+
+
+@pytest.fixture(autouse=True)
+def _mock_requests(monkeypatch) -> None:
+    """Mock requests so we don't have to run APIs."""
+
+    def mock_get(url: str, params: dict, *args: Any, **kwargs: Any) -> MockResponse:
+        logger.info("Mocking request for %s: %s", url, params)
```

Comment:
also when changing to loguru, will need to use `{}` style formatting instead of `%`